### PR TITLE
Update FindReferences baseline

### PR DESCRIPTION
**Pull request by MikhailArkhipov**
- Following changes to the 1/0 based location output.
- Move reference tests to a separate class (part of https://github.com/microsoft/python-language-server/issues/225)

### CODE REVIEW

File: src/Analysis/Engine/Test/AnalysisTest.cs [hunk 1]
```diff
@@ -6842,617 +5406,617 @@ print cls.g
...
-
-fob_2 = partial(fob, d = [], c = 'abc', b = 3.14, a = 123)
-result_2 = fob_2()
-
-fob_3 = partial(fob, 123, 3.14)
-result_3 = fob_3('abc', [])
-
-fob_4 = partial(fob, c = 'abc', d = [])
-result_4 = fob_4(123, 3.14)
+        /*
```
Comment (AlexanderSher) [inline src/Analysis/Engine/Test/AnalysisTest.cs]: Formatting offset

File: src/Analysis/Engine/Test/ReferencesTests.cs [hunk 2]
```diff
@@ -0,0 +1,1481 @@
...
+using Microsoft.PythonTools.Analysis;
+using Microsoft.PythonTools.Analysis.FluentAssertions;
+using Microsoft.PythonTools.Analysis.Values;
+using Microsoft.PythonTools.Interpreter;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using TestUtilities;
+
+namespace AnalysisTests {
+    [TestClass]
+    public class ReferencesTests: ServerBasedTest {
```
Comment (AlexanderSher) [inline src/Analysis/Engine/Test/ReferencesTests.cs]: `FindReferencesTests`?

File: src/Analysis/Engine/Test/LanguageServerTests.cs [hunk 3]
```diff
@@ -738,14 +738,14 @@ class D:
 
             // f
             var expected = new[] {
-                "Definition;(2, 5) - (2, 6)",
-                "Value;(2, 1) - (3, 11)",
-                "Reference;(5, 1) - (5, 2)",
-                "Reference;(10, 1) - (10, 2)",
-                "Reference;(17, 6) - (17, 7)"
+                "Definition;(1, 4) - (1, 5)",
```
Comment (AlexanderSher) [inline src/Analysis/Engine/Test/LanguageServerTests.cs]: Should we move this one to the references test class as well?

Comment (MikhailArkhipov) [reply src/Analysis/Engine/Test/LanguageServerTests.cs]: Probably... I guess it is not worth keeping LS tests separate since it is all about LS.

Comment (AlexanderSher) [reply src/Analysis/Engine/Test/LanguageServerTests.cs]: Ok.

File: src/Analysis/Engine/Test/ReferencesTests.cs (hunk 2 above)
Comment (MikhailArkhipov) [reply src/Analysis/Engine/Test/ReferencesTests.cs]: OK

**Pull request merged**